lms/static/js/student_profile/views/learner_profile_view.js

```diff
@@ -47,6 +45,11 @@
 
                 this.$('.profile-section-one-fields').append(this.options.usernameFieldView.render().el);
 
+                var imageView = this.options.profileImageFieldView;
+                imageView.undelegateEvents();
+                this.$('.profile-image-field').append(imageView.render().el);
```

Comment:
Nit: I'm never sure when you have to manually undelegate events. I *think* that Backbone is smart enough to do it for itself if you use it correctly. Check out the two top rated answers here:

http://stackoverflow.com/questions/9271507/how-to-render-and-append-sub-views-in-backbone-js